src/contractor/contractor.cpp

```diff
@@ -495,11 +495,13 @@ std::size_t Contractor::LoadEdgeExpandedGraph(
 
         // vector to count used speeds for logging
         // size offset by one since index 0 is used for speeds not from external file
-        std::vector<std::atomic<std::uint64_t>> segment_speeds_counters(
-            segment_speed_filenames.size() + 1);
-        for (auto &each : segment_speeds_counters)
-            each.store(0);
-        const constexpr auto LUA_SOURCE = 0;
+        std::vector<std::atomic<std::uint64_t>> segment_speeds_counters;
+        for (std::size_t i=0;i<segment_speed_filenames.size() + 1;++i)
+       {
+            segment_speeds_counters.emplace_back();
+            segment_speeds_counters[i].store(0);
+        }
```

Comment:
clang35's stdlib does not support the vector constructor that takes a size when the template argument is a moveable only type. This is a bug in its stdlib. To work around this we default construct the vector, then loop `n` times adding a new counter and initializing it explicitly.